docs/user-guide/setting-up-ci-cd.rst

```diff
@@ -0,0 +1,212 @@
+:desc: Set up a CI/CD pipeline to ensure that iterative improvements are tested and deployed with minimum manual effort
+
+.. _setting-up-ci-cd:
+
+Setting up CI/CD
+================
+
+Developing a bot looks a little different than developing a traditional
+software application, but that doesn't mean you should abandon software
+development best practices. Setting up CI/CD ensures that incremental updates
+to your bot are improving it, not harming it.
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+Continous Integration (CI) is the practice of merging in code changes
+frequently, and automatically testing changes as they are committed. Continuous
+Deployment (CD) means automatically deploying integrated changes to a staging
+or production environment. Together, they allow for more frequent improvements
+to your assistant with less manual effort expended on testing and deployment.
+
+This guide will cover **what** should go in a CI/CD pipeline, specific to a
+Rasa project. **How** you implement that pipeline is up to you. There are many
+CI/CD tools out there, and you might already have a preferred one. Many Git repository 
```

Comment:
> There are many CI/CD tools out there, and you might already have a preferred one. Many Git repository hosting services like Github, Gitlab, and Bitbucket provide their own CI/CD tools. It is recommended to incorporate your CI/CD pipeline into a Git repository set up, instead of using a seperate service.

Maybe something more like this...?
> There are many CI/CD tools out there: [GitHub Actions](https://github.com/features/actions), [GitLab CI/CD](https://docs.gitlab.com/ee/ci/), [Jenkins](https://jenkins.io/doc/), [CircleCI](https://circleci.com/docs/2.0/), etc. Selecting one that works well with your remote Git repository is a good idea.